AnnService/inc/Core/KDT/Index.h

```diff
@@ -148,8 +157,7 @@ namespace SPTAG
             ErrorCode SetParameter(const char* p_param, const char* p_value);
             std::string GetParameter(const char* p_param) const;
 
-            ErrorCode RefineIndex(const std::string& p_folderPath);
-            ErrorCode RefineIndex(const std::vector<std::ostream*>& p_indexStreams);
+            ErrorCode RefineIndex(const std::vector<std::ostream*>& p_indexStreams, bool* abort);
```

Comment:
please refer this interface.
https://msasg.visualstudio.com/Bing_and_IPG/_search?action=contents&text=ShouldAbort&type=code&lp=code-Project&filters=ProjectFilters%7BBing_and_IPG%7DRepositoryFilters%7BObjectStore%7D&pageSize=25&result=DefaultCollection%2FBing_and_IPG%2FObjectStore%2FGBmaster%2F%2Fprivate%2FPegasus%2Fsrc%2FANNPointLocalStore%2Fsrc%2FCommon%2FIAbortANNSaveMerge.h

        class IAbortANNSaveMerge
        {
        public:
            virtual bool ShouldAbort() = 0;
        };

and how hnsw refer the abort signal by macro AbortMergeSave in 

https://msasg.visualstudio.com/Bing_and_IPG/_git/ObjectStore?path=%2Fprivate%2FPegasus%2Fsrc%2FDssmSearch%2Fhnswnativelibrary%2Flib%2Fhnsw.cpp&_a=contents&version=GBmaster